test/cmdlineTests/asm_json_yul_export_evm_asm_import/args

```diff
@@ -0,0 +1 @@
+--import-asm-json - --opcodes --asm --bin --asm-json --pretty-json --json-indent 4
```

Comment:
Looks like this feature is also affected by #14628. The test output does not contain the non-JSON version of the assembly, even though you requested it.

Reply:
oki.. I would say we ignore that in this PR and then we fix it with the still open one

Reply:
Sure. Just please add a link to this PR from the issue saying that this needs to be fixed in assembler mode as well so that we don't forget.